### PR TITLE
V10 to V11

### PR DESCRIPTION
**Pull request by saif-serpentcs**
Migrated v10 to v11.
Facing some issue in Mysql and Purchase order in Vtiger.
Modules are under testing v11.

Will create a new Database in vTiger and test the modules, after that work on Travis.

### CODE REVIEW

File: vtiger_connector_calendar/__manifest__.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
+
 {
-    'name': 'Calendar VTiger Integration',
-    'version': '10.0.1.0.0',
-    'summary': 'Calendar VTiger Integration',
-    'description': """
-        Calendar VTiger Integration""",
+    'name': 'VTiger Base Connector Calender',
```
Comment (JayVora-SerpentCS) [inline vtiger_connector_calendar/__manifest__.py]: BASE nathi aa.
This module depends on base... its a VTiger Calendar Integration

File: vtiger_connector_calendar/__manifest__.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+    'version': '11.0.1.0.0',
+    'category': '',
```
Comment (JayVora-SerpentCS) [inline vtiger_connector_calendar/__manifest__.py]: why category is empty ?  
Category is like Gender

File: vtiger_connector_calendar/models/res_company.py [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
 
 import json
-import urllib
-import urllib2
+from odoo import api, models
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT as DT
+from datetime import datetime
+from datetime import timedelta
+from urllib.request import urlopen
+from urllib.request import Request
```
Comment (JayVora-SerpentCS) [inline vtiger_connector_calendar/models/res_company.py]: use combined import
from x import y,z

File: vtiger_connector_crm/__manifest__.py [hunk 4]
```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
+
 {
-    'name': 'CRM VTiger Integration',
-    'version': '10.0.1.0.0',
-    'summary': 'CRM VTiger Integration',
-    'description': """
-        CRM VTiger Integration""",
+    'name': 'VTiger Base Connector Lead',
```
Comment (JayVora-SerpentCS) [inline vtiger_connector_crm/__manifest__.py]: module name and category badhe correct karo

File: vtiger_connector_calendar/__manifest__.py (hunk 1 above)
Comment (saif-serpentcs) [reply vtiger_connector_calendar/__manifest__.py]: Yes, Yes, okay sir, I will correct it.
Thanks

File: vtiger_connector_calendar/__manifest__.py (hunk 2 above)
Comment (saif-serpentcs) [reply vtiger_connector_calendar/__manifest__.py]: ok sir, will add it.

File: vtiger_connector_calendar/models/res_company.py (hunk 3 above)
Comment (saif-serpentcs) [reply vtiger_connector_calendar/models/res_company.py]: ok sir, will do it.

File: vtiger_connector_crm/__manifest__.py (hunk 4 above)
Comment (saif-serpentcs) [reply vtiger_connector_crm/__manifest__.py]: ok

File: vtiger_connector_calendar/__manifest__.py (hunk 2 above)
Comment (saif-serpentcs) [reply vtiger_connector_calendar/__manifest__.py]: **Done**

File: vtiger_connector_calendar/models/res_company.py (hunk 3 above)
Comment (saif-serpentcs) [reply vtiger_connector_calendar/models/res_company.py]: **Done**

File: vtiger_connector_crm/__manifest__.py (hunk 4 above)
Comment (saif-serpentcs) [reply vtiger_connector_crm/__manifest__.py]: **Done**

File: vtiger_connector_base/__manifest__.py [hunk 5]
```diff
@@ -1,20 +1,17 @@
...
+# See LICENSE file for full copyright and licensing details.
+
 {
     'name': 'VTiger Base Connector',
-    'version': '10.0.1.0.0',
-    'summary': 'VTiger Base Connector',
-    'description': """
-        VTiger Base Connector""",
+    'version': '11.0.1.0.0',
+    'category': 'Hidden',
```
Comment (JayVora-SerpentCS) [inline vtiger_connector_base/__manifest__.py]: kem Hidden ? Should not it be extra tools?
@saif-serpentcs kem aam dost?  logic nai lagavvanu?

File: vtiger_connector_base/__manifest__.py [hunk 6]
```diff
@@ -1,20 +1,17 @@
...
-    ],
-    'data': [
-        'views/vtiger_templates.xml',
-        'data/vtiger_connector_base_data.xml',
-        'views/res_company_view.xml'
-    ],
+    'depends': ['base'],
+    'data': ['views/vtiger_templates.xml',
+             'data/vtiger_connector_base_data.xml',
+             'views/res_company_view.xml'],
```
Comment (JayVora-SerpentCS) [inline vtiger_connector_base/__manifest__.py:15]: logical sequence ma data first, then view files.

File: vtiger_connector_calendar/__manifest__.py [hunk 7]
```diff
@@ -1,20 +1,17 @@
...
-    'depends': [
-        'vtiger_connector_base',
-        'calendar',
-    ],
-    'data': [
-        'views/res_company_view.xml',
-        'views/calendar_views.xml',
-    ],
+    'depends': ['vtiger_connector_base',
+                'calendar'],
```
Comment (JayVora-SerpentCS) [inline vtiger_connector_calendar/__manifest__.py]: always odoo module pehla ave.

File: vtiger_connector_base/__manifest__.py (hunk 5 above)
Comment (saif-serpentcs) [reply vtiger_connector_base/__manifest__.py]: @JayVora-SerpentCS can i add **Generic Module** or **Base Module**?

**Comment by saif-serpentcs** [comment]
@JayVora-SerpentCS 
Could you please review and merge it v11?
Thanks.

**Pull request merged**